ragondin/routers/indexer.py

```diff
@@ -86,44 +75,154 @@ async def search(query_params: SearchRequest, indexer: Indexer = Depends(get_ind
         raise HTTPException(status_code=500, detail=str(e))
 
 
-@router.delete("/delete-files/",response_model=None)
-async def delete_files(request : DeleteFilesRequest, indexer: Indexer = Depends(get_indexer)):
-    """
-    Delete points in Vector DB associated with the given file names.
+@router.get("/{partition}", response_model=None)
```

Comment:
Same remark about the GET for a search: a `POST /{partition}/search` or `POST /{user_id}/search` might make more sense